packages/tokens/config-provider.ts

```diff
@@ -0,0 +1,7 @@
+import type { configProviderProps } from '@element-plus/components/config-provider/config-provider'
```

Comment:
I think we are creating circular referencing here since `@element-plus/hooks` references this file and this file references `@element-plus/components`, `@element-plus/components` references this file.

Can we pull `button` out into this file and import it from here?

```ts
export const globalConfigProps = { 
  buildProp({
    type: definePropType<ButtonConfigContext>(Object),
    default: () => {
      return mutable({
        autoInsertSpace: true,
      } as const)
    },
  }),
}
```

Reply:
Then use it with 

> packages/components/config-provider/config-provider.ts

```ts
export const configProviderProps = {
  ...useLocaleProps,
  ...useGlobalConfigProps,
}
```

Reply:
I have reorganized the code according to the directory structure of our components